save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -74,18 +77,36 @@ class VulnerabilityController(
         ResponseEntity.ok("Vulnerability was successfully saved")
     }
 
-    @PostMapping("/save-projects")
+    @PostMapping("/update")
     @Operation(
         method = "POST",
         summary = "Update vulnerability.",
         description = "Update vulnerability.",
     )
     @ApiResponse(responseCode = "200", description = "Successfully updated vulnerability")
+    fun update(
+        @RequestBody vulnerabilityDto: VulnerabilityDto,
+        authentication: Authentication,
+    ): Mono<StringResponse> = blockingToMono {
+        if (authentication.hasRole(Role.SUPER_ADMIN)) {
```

Comment:
should be return 403 in case when use is not SUPER_ADMIN?

Reply:
Maybe do it with `@PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")`?

Reply:
Fixed